### PR TITLE
Adding tip 74

### PR DESCRIPTION
**Pull request by pr-Mais**

### CODE REVIEW

**Comment by bizz84** [comment]
@pr-Mais could you remove the `pubspec` files?
These tips are just meant to show some code snippets, not entire runnable projects.

File: tips/0074-pass-functions-as-arguments/pubspec.lock [hunk 1]
```diff
@@ -0,0 +1,153 @@
+# Generated by pub
```
Comment (bizz84) [inline tips/0074-pass-functions-as-arguments/pubspec.lock:1]: Please remove this file

File: tips/0074-pass-functions-as-arguments/pubspec.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name: send_account_updates_with_twilio
```
Comment (bizz84) [inline tips/0074-pass-functions-as-arguments/pubspec.yaml:1]: Please remove this file.

**Review by bizz84 — CHANGES_REQUESTED** [review]

**Comment by pr-Mais** [comment]
Done

**Pull request merged**